packages/aws-cdk/lib/convert-to-cli-args.ts

```diff
@@ -254,3 +254,197 @@ export function convertToCliArgs(args: any): CliArguments {
+    toolkitStackName: args.deploy?.toolkitStackName,
+    progress: args.deploy?.progress,
+    rollback: args.deploy?.rollback,
+    hotswap: args.deploy?.hotswap,
+    hotswapFallback: args.deploy?.hotswapFallback,
+    watch: args.deploy?.watch,
+    logs: args.deploy?.logs,
+    concurrency: args.deploy?.concurrency,
+    assetParallelism: args.deploy?.assetParallelism,
+    assetPrebuild: args.deploy?.assetPrebuild,
+    ignoreNoStacks: args.deploy?.ignoreNoStacks,
+  };
+  const rollbackOptions = {
+    all: args.rollback?.all,
+    toolkitStackName: args.rollback?.toolkitStackName,
+    force: args.rollback?.force,
+    validateBootstrapVersion: args.rollback?.validateBootstrapVersion,
+    orphan: args.rollback?.orphan,
+  };
+  const importOptions = {
+    execute: args.import?.execute,
+    changeSetName: args.import?.changeSetName,
+    toolkitStackName: args.import?.toolkitStackName,
+    rollback: args.import?.rollback,
+    force: args.import?.force,
+    recordResourceMapping: args.import?.recordResourceMapping,
+    resourceMapping: args.import?.resourceMapping,
+  };
+  const watchOptions = {
+    buildExclude: args.watch?.buildExclude,
+    exclusively: args.watch?.exclusively,
+    changeSetName: args.watch?.changeSetName,
+    force: args.watch?.force,
+    toolkitStackName: args.watch?.toolkitStackName,
+    progress: args.watch?.progress,
+    rollback: args.watch?.rollback,
+    hotswap: args.watch?.hotswap,
+    hotswapFallback: args.watch?.hotswapFallback,
+    logs: args.watch?.logs,
+    concurrency: args.watch?.concurrency,
+  };
+  const destroyOptions = {
+    all: args.destroy?.all,
+    exclusively: args.destroy?.exclusively,
+    force: args.destroy?.force,
+  };
+  const diffOptions = {
+    exclusively: args.diff?.exclusively,
+    contextLines: args.diff?.contextLines,
+    template: args.diff?.template,
+    strict: args.diff?.strict,
+    securityOnly: args.diff?.securityOnly,
+    fail: args.diff?.fail,
+    processed: args.diff?.processed,
+    quiet: args.diff?.quiet,
+    changeSet: args.diff?.changeSet,
+  };
+  const metadataOptions = {};
+  const acknowledgeOptions = {};
+  const noticesOptions = {
+    unacknowledged: args.notices?.unacknowledged,
+  };
+  const initOptions = {
+    language: args.init?.language,
+    list: args.init?.list,
+    generateOnly: args.init?.generateOnly,
+  };
+  const migrateOptions = {
+    stackName: args.migrate?.stackName,
+    language: args.migrate?.language,
+    account: args.migrate?.account,
+    region: args.migrate?.region,
+    fromPath: args.migrate?.fromPath,
+    fromStack: args.migrate?.fromStack,
+    outputPath: args.migrate?.outputPath,
+    fromScan: args.migrate?.fromScan,
+    filter: args.migrate?.filter,
+    compress: args.migrate?.compress,
+  };
+  const contextOptions = {
+    reset: args.context?.reset,
+    force: args.context?.force,
+    clear: args.context?.clear,
+  };
+  const docsOptions = {
+    browser: args.docs?.browser,
```

Comment:
do we want to allow aliasing in the `cdk.json` file? if so, this becomes incomplete... because `doc` is valid also. i think we shouldn't

Reply:
Let's avoid it if we can